### PR TITLE
HIVE-23590 stale prs

### PR DESCRIPTION
**Pull request by kgyrtkirk**

### CODE REVIEW

**Comment by kgyrtkirk** [comment]
I killed the second test execution - updates to asf.yaml and github actions are not covered by hive tests

File: .github/workflows/stale.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/stale@v3
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        stale-pr-message: 'This pull request has been automatically marked as stale because it has not had recent activity. It will be closed if no further activity occurs.'
+        exempt-pr-labels: 'awaiting-approval,work-in-progress'
+        stale-pr-label: stale
```
Comment (jcamachor) [inline .github/workflows/stale.yml:15]: @kgyrtkirk , should we use the `days-before-stale` and `days-before-close` configs? Maybe closing them automatically after 7 days?

Comment (kgyrtkirk) [reply .github/workflows/stale.yml:15]: I thinked that we could use the defaults which are 60/7 respectively
https://github.com/actions/stale/blob/master/action.yml
should I add them to make it clearer?

Comment (jcamachor) [reply .github/workflows/stale.yml:15]: Oh, I did not realize that. Yes, I think making them explicit may be better.

**Pull request merged**